pages/loginPage.js

```diff
@@ -0,0 +1,41 @@
+const { Page } = require('./page');
+
+const usernameInput = '#username';
+const passwordInput = '#password';
+const loginButton = 'input[name="login"]';
+const flashNotice = '#flash_notice';
+const flashError = '#flash_error';
+
+class LoginPage extends Page {
+    constructor(page) {
+        super(page);
+    }
+
+    async fillUsername(username) {
+        await this.page.locator(usernameInput).fill(username);
+    }
+
+    async fillPassword(password) {
+        await this.page.locator(passwordInput).fill(password);
+    }
+
+    async submit() {
```

Comment:
@karenn32 The name of the method is not clear, please, use instead 'clickLoginButton' or something like that.